common/src/stack/command/stack/commands/add/pallet/plugin_remote_path.py

```diff
@@ -70,6 +70,6 @@ def run(self, args):
 			try:
 				result = self.owner._exec(wget_cmd, check=True)
 			except subprocess.CalledProcessError as e:
-				raise CommandError(self, f'remote fetch failed, tried:\n{" ".join(wget_cmd)}: {e.stderr}')
+				raise CommandError(self, f'remote fetch failed, tried:\n{" ".join(wget_cmd)}\n{e.stderr}')
```

Comment:
Consider also providing `stdout` in the error message. It's better to provide more information than not enough.